### PR TITLE
gh-91340: Document multiprocessing.set_start_method force parameter

### PR DESCRIPTION
**Pull request by dignissimus**
#91340

https://bugs.python.org/issue47184

Automerge-Triggered-By: GH:kumaraditya303

### CODE REVIEW

**Comment by MaxwellDupre** [comment]
Where you have
*method* can be ``'fork'``, ``'spawn'`` or ``'forkserver'``.
The start of the sentence is a lower case italic (in the middle of a line of text), but it looks strange: optionally start new line or make more distinctive, please.

**Comment by kumaraditya303** [comment]
You can remove the news entry, it's not needed.

File: Doc/library/multiprocessing.rst [hunk 1]
```diff
@@ -1078,10 +1078,14 @@ Miscellaneous
...
       Accepts a :term:`path-like object`.
 
-.. function:: set_start_method(method)
+.. function:: set_start_method(method, force=False)
 
    Set the method which should be used to start child processes.
-   *method* can be ``'fork'``, ``'spawn'`` or ``'forkserver'``.
+   The *method* argument can be ``'fork'``, ``'spawn'`` or ``'forkserver'``.
+   Raises :exc:`RuntimeError` if the start method has already been set and *force*
+   is not set to ``True``.  If *method* is ``None`` and *force* is set to ``True``
```
Comment (kumaraditya303) [inline Doc/library/multiprocessing.rst:1086]:
```suggestion
   is ``False``.  If *method* is ``None`` and *force* is set to ``True``
```

**Review by kumaraditya303 — APPROVED** [review]

File: Doc/library/multiprocessing.rst [hunk 2]
```diff
@@ -1078,10 +1078,14 @@ Miscellaneous
...
+   then the start method is set to ``None``.  If *method* is ``None`` and *force* is
+   set to ``False`` then the context is set to the default context.
```
Comment (kumaraditya303) [inline Doc/library/multiprocessing.rst]:
```suggestion
   The *method* argument can be ``'fork'``, ``'spawn'`` or ``'forkserver'``.
   Raises :exc:`RuntimeError` if the start method has already been set and *force*
   is not ``True``.  If *method* is ``None`` and *force* is ``True`` then the start
   method is set to ``None``.  If *method* is ``None`` and *force* is ``False``
   then the context is set to the default context.
```

**Comment by kumaraditya303** [comment]
Thanks.

**Pull request merged**